tools/deb/BUILD

```diff
@@ -0,0 +1,52 @@
+load("@bazel_tools//tools/build_defs/pkg:pkg.bzl", "pkg_tar", "pkg_deb")
+
+pkg_tar(
+    name = "envoy-bin",
+    files = [
+        "istio-iptables.sh",
+        "istio-start.sh",
+        "//src/envoy/mixer:envoy",
+    ],
+    mode = "0755",
+    package_dir = "/usr/local/bin",
+)
+
+pkg_tar(
+    name = "istio-conf",
+    files = [
+        "envoy.json",
+        "sidecar.env",
+    ],
+    mode = "0755",
+    package_dir = "/var/lib/istio/envoy",
+)
+
+pkg_tar(
+    name = "istio-systemd",
+    files = ["istio.service"],
+    mode = "644",
+    package_dir = "/lib/systemd/system",
+)
+
+pkg_tar(
+    name = "debian-data",
+    extension = "tar.gz",
+    deps = [
+        ":envoy-bin",
+        ":istio-conf",
+        ":istio-systemd",
+    ],
+)
+
+pkg_deb(
+    name = "istio-proxy",
```

Comment:
Do you mind istio-envoy ? It would be nice for all istio- related debs to be close to each other in 
"dpkg -l". 

Technically we should have a meta-package istio-proxy, with possible selections - but don't know how to do that, and right now we only have envoy. My only concern is that a user may not know
what 'envoy' is - istio-proxy-envoy may at least let user guess it's the proxy/sidecar.

I'm fine either way - but since the name is already used in scripts, let's decide on one color :-)